src/examples/example-do-do-not.tsx

```
@@ -0,0 +1,76 @@
+    private renderSection(
+        type: string,
+        iconName: string,
+        title: string,
+        description?: string,
+    ) {
+        return (
+            <section class={type}>
+                <div>
+                    <slot name={type} />
+                    <footer>
+                        <limel-icon name={iconName} aria-hidden="true" />
```

Biome (1.9.4)

[error] Disallow aria-hidden="true" from being set on focusable elements.

aria-hidden should not be set to true on focusable elements because this can lead to confusing behavior for screen reader users.
Unsafe fix: Remove the aria-hidden attribute from the element.


(lint/a11y/noAriaHiddenOnFocusable)